packages/tags/src/TagsEditor.tsx

```diff
@@ -137,7 +145,10 @@ export function TagsEditor(props: TagsEditorProps) {
               label={item}
               index={index}
               key={item + index}
+              // disabled goes for SortableElement
               disabled={isDisabled}
+              // isDisabled goes for SortablePill
+              isSortablePillDisabled={isDisabled}
```

Comment:
Any reason why a single `disabled` prop won't work? I think we always want the `SortablePill` to be disabled & its children, no?

Reply:
`<SortablePill />` is a component that uses [SortableElement](https://github.com/clauderic/react-sortable-hoc#sortableelement-hoc) HOC, which will not pass the disabled props down to the `<Pill />` component.


```jsx
const SortablePill = SortableElement((props: SortablePillProps) => (
 // props.disabled is not available here
  <Pill
    testId="tag-editor-pill"
    className={cx(styles.pill, { [styles.pillDisabled]: props.isSortablePillDisabled })}
    label={props.label}
    onClose={() => {
      if (!props.isSortablePillDisabled) {
        props.onRemove(props.index);
      }
    }}
    onDrag={noop}
    dragHandleComponent={<SortablePillHandle isDisabled={props.isSortablePillDisabled} />}
  />
));
```

Reply:
Thanks for the explanation. That's unfortunate but it's Ok I guess since the `SortablePill` isn't exposed outside the package. I think it's worth noting that down though as a code comment as it's not very obvious at the moment.

Reply:
done